locations/Bahraich/inputs/generation/grid_inputs.yaml

```diff
@@ -0,0 +1,74 @@
+---
+################################################################################
+# grid_inputs.yaml - Grid parameters.                                          #
+#                                                                              #
+# Author: Paul Harfouche                                                       #
+# Copyright: Phil Sandwell, Ben Winchester & Paul Harfouche 2022               #
+# Date created: 01/06/2022                                                     #
+# License: Open source                                                         #
+################################################################################
+
+exchange_rate: 25000 #[LBP/$]
+
+grid:
+# Diesel Generator (split into 5A and 10A subcriptions)
```

Comment:
## Specific comment
See `Yamllint` comment in the file here - indent isn't quite right in the code.

## General point 👉 
The `Bahraich` location is designed as a ready-to-go default, which isn't too specific. So, this file looks good for your use, but, generally, we probably want to simplify it at some point 😄

Reply:
@paulharfouche , this still needs resolving, see the in-file comment that `yamllint` has put here, or run `yamllint` by hand :smile: